dev/tools/tokens/lib/fab_template.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'template.dart';
+
+class FABTemplate extends TokenTemplate {
+  FABTemplate(String fileName, Map<String, dynamic> tokens) : super(fileName, tokens);
```

Comment:
chore: if you made the superclass constructor `const`, this may be `const` as well

```suggestion
  const FABTemplate(
    String fileName, 
    Map<String, dynamic> tokens,
  ) : super(fileName, tokens);
```